testgrid/specs/os-latest.yaml

```diff
@@ -29,6 +29,10 @@
     curl -L -o centos2ol.sh https://raw.githubusercontent.com/oracle/centos2ol/main/centos2ol.sh
     chmod +x centos2ol.sh
     bash centos2ol.sh -r
+- id: rocky-88
```

Comment:
should you remove centos-84 from os-latest?

Reply:
Agreed I would love this if we removed centos 8.4